riocli/secret/delete.py

```diff
@@ -41,8 +41,8 @@ def delete_secret(force: str, secret_name: str, secret_guid: str, spinner=None)
                 abort=True)
 
     try:
-        client = new_client()
-        client.delete_secret(secret_guid)
+        client = new_v2_client(with_project=True)
+        client.delete_secret(secret_name)
         spinner.text = click.style('Secret deleted successfully.', fg=Colors.GREEN)
         spinner.green.ok(Symbols.SUCCESS)
     except Exception as e:
```

Comment:
The output of the failure should be show a proper failure message with symbol.
03:40:17 pallab@pop-os ~ 
→ rio secret delete docker01
secrets: secret not found
```